frontend/angular.json

```diff
@@ -46,7 +46,7 @@
                         "tsConfig": "tsconfig.app.json",
                         "inlineStyleLanguage": "sass",
                         "assets": ["src/favicon.ico", "src/assets"],
-                        "styles": ["src/styles/index.sass"],
+                        "styles": ["src/styles/index.sass", "node_modules/ngx-toastr/toastr.css"],
```

Comment:
Please, don't do it
No node_modules in project

Reply:
so I need to import it form another place?